src/librustc_mir/borrow_check/nll/type_check/mod.rs

```diff
@@ -1112,6 +1112,17 @@ impl<'a, 'gcx, 'tcx> TypeChecker<'a, 'gcx, 'tcx> {
                             .constraints
                             .liveness_constraints
                             .add_element(region_vid, term_location);
+
+                        if let Some(facts) = borrowck_context.all_facts {
```

Comment:
These facts I believe are not needed in polonius — pretty sure I excluded them intentionally. We only care about the "transitive relations" that are formed via these late-bound regions. Did you find they were needed in particular?